lib/utils/compose.js

```diff
@@ -241,7 +241,6 @@ function originalTarDirectory(dir, param) {
 	}
 
 	const getFiles = () =>
-		// @ts-ignore `klaw` returns a `Walker` which is close enough to a stream to work but ts complains
```

Comment:
Turns out the real issue was a bug in the types, updating the types to pick up that fix lets us remove this hack :)